py/desispec/scripts/healpix_redshifts.py

```diff
@@ -18,21 +18,21 @@ def parse(options=None):
     import argparse
 
     p = argparse.ArgumentParser()
-    p.add_argument('--healpix', type=str, required=False,
-            help='nested healpix numbers to run (comma separated)')
     p.add_argument('--survey', type=str, required=True,
             help='survey (e.g. sv3, main)')
     p.add_argument('--program', type=str, required=True,
             help='program (e.g. dark, bright, backup)')
-    p.add_argument('--nside', type=int, default=64,
```

Comment:
Yes.  In current main, `args.nside` is used to set a header keyword but not actually used to change what nside is used (bug!).  Given that we've never used anything except nside=64 anyway, I didn't want to get distracted in this PR by fixing and testing that, so I just went back to hard coding nside=64 and removing the option that implied that it could be changed.  If we find that we do need something different in the future, we'll need a deeper dive on how to keep all the pieces consistent.